internal/util/paramtable/base_param.go

```diff
@@ -29,90 +29,234 @@ import (
 // embedding BaseTable. It is used to quickly and easily access the system configuration.
 type BaseParamTable struct {
 	BaseTable
-	once sync.Once
 
-	// --- ETCD ---
-	EtcdEndpoints []string
-	MetaRootPath  string
-	KvRootPath    string
-
-	// --- Embed ETCD ---
-	UseEmbedEtcd   bool
-	EtcdConfigPath string
-	EtcdDataDir    string
+	EtcdCfg    EtcdConfig
+	PulsarCfg  pulsarConfig
```

Comment:
PulsarConfig, so is MinioConfig and RocksMqConfig

Reply:
done